api/src/repositories/categories.ts

```diff
@@ -11,6 +11,11 @@ class CategoriesRepository {
 
         return category;
     }
+
+    async findAll() {
+        const categories = await prisma.category.findMany();
+        return categories;
+    }
```

Comment:
```suggestion
    async findAll() {
        const categories = await prisma.category.findMany();
        
        return categories;
    }
```

Vamos tentar sempre deixar a linha do return isolada